### ISSUE TITLE
Take 6 Online Multiplayer issues

### ISSUE THREAD

**Issue by Yannick-S** [freeboardgames/FreeBoardGames.org#482]
Hi,
we tried to play Take6! as a group of 5 people. Login and Setup worked well. But once the first player plays the first card, all other players recieve "Waiting for opponenets ... " and can no longer play cards. The game is thus stuck after any player plays the first card.

(Otherwise thanks! This whole stuff is amazing)

**Comment by flamecoals** [comment]
Hi @Yannick-S and thank you for your feedback!

How many days ago did you try this? Sadly after we upgrade boargame.io we got a very similar bug (#463) that went unnoticed for 1 week in production, because our tests were not running. 

We fixed the game and tests a couple of days ago, could you try again? I will try playing with 5 here with my devices (laptop, phone, etc)...

**Comment by flamecoals** [comment]
I just play tested a game here with 5 browsers and it worked :), so I believe you were a victim of #463 ... Sorry for that, I know it must be really frustrating to manage to arrange so many people to play and then teh game is broken :/. I hope this does not happen again because we re-activated our tests.

![image](https://user-images.githubusercontent.com/1034631/78924783-3a756a80-7a4f-11ea-83e7-56eee6b04235.png)


With that said, while play testing I noticed the lack of context for most people who's turn is it. I think this should be fairly easy to do (we already have this information piped to all clients), so I created #483 to add this visual cue of who's turn is it and who should be playing.
